apps/website/src/examples/Textarea.scroll.tsx

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Label, Textarea } from '@itwin/itwinui-react';
+
+export default () => {
+  const [value, setValue] = React.useState(
+    'If the area is shrunk to a size smaller than the amount of text within, a scrollbar appears to keep navigation possible. This behavior is especially important for text areas without a resizing handle, to avoid making text editing tedious.'
```

Comment:
It seems a bit repetitive to me to have this explanation inside the text box and outside of it, instead of having filler text inside the box. I'm curious why you did it like this?

Reply:
I don't think anyone reads what's inside the example.

I can update that to lorem ipsum or some other random text, if you prefer that way.

Reply:
That makes sense; I was looking at it really closely to review it.